sacra-scripta.csl

```diff
@@ -0,0 +1,511 @@
+        <text variable="title" quotes="true" font-style="normal"/>
+      </else-if>
+      <else-if type="article" match="any"/>
+      <else>
+        <text variable="title" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title-short">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" font-style="italic" form="short"/>
+      </if>
+      <else>
+        <text variable="title" quotes="true" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", ">
+      <text variable="publisher"/>
+      <text variable="publisher-place"/>
+    </group>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if match="any" variable="edition">
+        <choose>
+          <if type="graphic report" match="any">
+            <date variable="issued" prefix=" ">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" " text-case="lowercase"/>
+              <date-part name="year"/>
+            </date>
+          </if>
+          <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis" match="any">
+            <date variable="issued">
+              <date-part name="year"/>
+            </date>
+          </else-if>
+          <else>
+            <date variable="issued">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" " text-case="lowercase"/>
+              <date-part name="year"/>
+            </date>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <choose>
+          <if type="graphic report" match="any">
+            <date variable="issued" prefix=",  ">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" " text-case="lowercase"/>
+              <date-part name="year"/>
+            </date>
+          </if>
+          <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference report song thesis" match="any">
+            <date variable="issued" prefix=", ">
+              <date-part name="year"/>
+            </date>
+          </else-if>
+          <else>
+            <date variable="issued" prefix=", ">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" " text-case="lowercase"/>
+              <date-part name="year"/>
+            </date>
+          </else>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <group font-style="normal" delimiter=" " prefix="coll. ">
+      <text variable="collection-title" font-style="italic"/>
+      <text variable="collection-number"/>
+    </group>
+  </macro>
+  <macro name="locators">
+    <group>
+      <label variable="locator" form="short"/>
+      <text variable="locator" prefix=" "/>
```

Comment:
It's recommended to avoid unnecessary pre/suffixes and better use groups using `delimiter=" "`.